charts/common/templates/_deployment.yaml

```diff
@@ -77,24 +77,24 @@ spec:
               path: {{ if .Values.livenessProbe -}} {{ default "/admin/health" .Values.livenessProbe.path | quote }} {{- else -}} "/admin/health" {{- end }}
               port: {{ .Values.service.containerPort }}
             periodSeconds: {{ if .Values.livenessProbe -}} {{ default 60 .Values.livenessProbe.periodSeconds }} {{- else -}} 60 {{- end }}
-            timeoutSeconds: {{ if .Values.livenessProbe -}} {{ default 10 .Values.livenessProbe.timeoutSeconds }} {{- else -}} 10 {{- end }}
+            timeoutSeconds: {{ if .Values.livenessProbe -}} {{ default 10 .Values.livenessProbe.timeoutSeconds }} {{- else -}} 30 {{- end }}
```

Comment:
In case we set as an example:
livenessProbe:
    path: ...
we will still have the timeoutSeconds equal to 10. You should also change the first argument of default function.